doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -446,6 +446,8 @@ Bluetooth libraries and services
     * The :kconfig:option:`CONFIG_BT_FAST_PAIR_SUBSEQUENT_PAIRING` Kconfig option allowing the user to control the support for the Fast Pair subsequent pairing feature.
     * The :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE` Kconfig choice option allowing the user to select their target Fast Pair use case.
       The :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_UNKNOWN`, :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_INPUT_DEVICE`, :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_LOCATOR_TAG` and :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_MOUSE` Kconfig options represent the supported use cases that can be selected as part of this Kconfig choice option.
+  * Dropped the MbedTLS cryptographic backend support in Fast Pair, because it could not be used with nRF54L SOCs and non-secure platforms.
```

Comment:
Perhaps this is a better argument (we don't want to maintain two backends that has nearly the same APIs) and I think there is plan to deprecate mbedTLS at some point

```suggestion
  * Dropped the MbedTLS cryptographic backend support in Fast Pair, because it is superseded by the PSA backend.
```